Cargo.toml

```diff
@@ -44,147 +44,149 @@ stout-runtime = { path = "runtime/stout" }
 jsonrpsee = { version = "0.16.2" }
```

Comment:
Need to update `clap`, `serde`, `hex-literal`, `scale-info` versions too.

Reply:
done